lms/djangoapps/courseware/toggles.py

```diff
@@ -61,3 +61,41 @@ def course_exit_page_is_active(course_key):
         REDIRECT_TO_COURSEWARE_MICROFRONTEND.is_enabled(course_key) and
         COURSEWARE_MICROFRONTEND_COURSE_EXIT_PAGE.is_enabled(course_key)
     )
+
+# .. toggle_name: courseware.mfe_progress_milestones
```

Comment:
Not saying to change it, but just offering another thought in case you didn't consider. 

I wonder if we should have all the flags at the top and the functions at the bottom. Or we could do it like it is here where the function is next to the flag it relates to. 

Totally up to you, but just bringing up both options.

Reply:
yeah I like functions at the bottom, this was just a lazy merge on my part